evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -34,7 +34,7 @@
                             <div class="vote-inputs {{ field.field.widget.attrs.choices.cssClass }} btn-group btn-group-toggle" data-toggle="buttons"{% if preview %} disabled{% endif %}>
                             {% for choice, color in field|zip:field.field.widget.attrs.choices.colors %}
                                 <label class="btn btn-sm vote-btn vote-btn-{{ color }} d-flex{% if field.value == choice.data.value %} active{% endif %}{% if field.errors %} choice-error{% endif %}" name="{{ choice.data.name }}"{% if not preview %} onclick="selectedAnswer('{{ choice.data.name }}');"{% endif %}>
-                                    <input id="{{ choice.id_for_label }}" name="{{ choice.data.name }}" type="radio" value="{{ choice.data.value }}" autocomplete="off"{% if field.value == choice.data.value %} checked{% endif %} />
+                                    <input class="tab-selectable num-selectable" id="{{ choice.id_for_label }}" name="{{ choice.data.name }}" type="radio" value="{{ choice.data.value }}" autocomplete="off"{% if field.value == choice.data.value %} checked{% endif %} />
```

Comment:
Please put `class` behind `id` and `name` (I’m not quite sure about the order of `class` and `name`, but at least `id` must come before `class`)

Reply:
I never heard of this, but I can stick to it here where I add the `class` attribute. Note that there is more code around the diff that doesn't comply to this rule, but was in here before this PR, should this be fixed?